components/compliance-service/api/tests/010_controls_list_spec.rb

```diff
@@ -0,0 +1,373 @@
+##### GRPC SETUP #####
+require 'api/reporting/reporting_pb'
+require 'api/reporting/reporting_services_pb'
+
+describe File.basename(__FILE__) do
+  Reporting = Chef::Automate::Domain::Compliance::Api::Reporting unless defined?(Reporting)
+
+  def reporting
+    Reporting::ReportingService;
+  end
+
+  # it "no filters gets them all filter" do
+  #   assert_grpc_error("rpc error: code = InvalidArgument desc = 'control_tag' filter is required for 'control_tag_value' suggestions", 2) do
```

Comment:
assuming you didn't intend to include the commented block here?

Reply:
good catch.. i'll clean that up